### PR TITLE
Fancy links for GH issues representing external reviews

### PR DESCRIPTION
**Pull request by jyasskin**
For all open issues, this adds the amount of time they've been open: 
image

For open position issues, colors the links according to how long they've been open, with thresholds at 2 weeks and 1 month. https://www.chromium.org/blink/launching-features/#new-feature-prepare-to-ship says to give reviewers at least a month to review, so ... @chrishtr @miketaylr what colors do you want for what ages?
image

For resolved position issues, colors the links according to the position:
image

### CODE REVIEW

File: client-src/elements/chromedash-link.js
Comment (jrobbins) [inline client-src/elements/chromedash-link.js:131]: We already have `<sl-relative-time>` as used in utils.js.  Could you use that? It might even just be wrapping the same underlying library.

**Review by jrobbins — APPROVED** [review]
LGTM after considering one possible change.

File: client-src/elements/chromedash-link.js
Comment (jyasskin) [reply client-src/elements/chromedash-link.js:131]: `<sl-relative-date>` is missing the `precision` argument that I'm using to turn "Opened 2 Hours Ago" into "Opened Today" (once https://github.com/github/relative-time-element/pull/280 goes in), but that's not a big deal if you'd rather avoid the dependency. I'll see if there are any other problems once my development machine finishes upgrading.

Comment (jyasskin) [reply client-src/elements/chromedash-link.js]: I have to undo the style from https://github.com/GoogleChrome/chromium-dashboard/blob/cffa57147e486fc1e44e972740306c461e884342/client-src/css/shared-css.js#L191-L193
but then it seems to work:
image

Here's a patch without `<relative-time>`.

**Comment by miketaylr** [comment]
> so ... @chrishtr @miketaylr what colors do you want for what ages?

Hmm, this is trickier to consider than I would hope for. I do see the ~one month guidance for TAG, but not for other vendors, FWIW. I think the colors you have do make sense, once we document them. Red: "probably too soon for a fulsome response", orange: "proceed with caution" and gray/black: "YOLO 💀". Did I get that right?

> For resolved position issues, colors the links according to the position:

This looks great.

**Comment by jyasskin** [comment]
@miketaylr That's about right. I'd s/YOLO/the API owners have to make a call/ though. ;) I'll send you a CL to copy the TAG's 1-month guideline to the other reviewers.

**Comment by chrishtr** [comment]
> https://www.chromium.org/blink/launching-features/#new-feature-prepare-to-ship now asks for 1 month for vendors too.

It isn't up to us how long other vendors or the TAG take to respond though. Is this meant to represent whether the issue itself is overdue, or something whether a feature owner should consider it "timed out"?

**Comment by jyasskin** [comment]
It's more about whether the external reviewer has clearly not had enough time to review it. The color here only goes to neutral gray after a month and not green because I don't think there's any automated way to decide whether the review has timed out. That's up to the human reviewers.

**Comment by miketaylr** [comment]
> It isn't up to us how long other vendors or the TAG take to respond though.

Correct. But I think guidance to ask early enough to improve the odds of getting a response is good. And then API OWNERs can make a call (as we sometimes do "let's give it a week or two to bake and hope for a response", etc.)

**Comment by chrishtr** [comment]
Ok makes aense. How about yellow for less than 1 months, meaning "in progress, shipping might block on it"? And after that gray.

**Comment by jyasskin** [comment]
Works for me. I'll make that change Monday.

**Comment by jyasskin** [comment]
Ok, everything less than a month is now yellow. I'll merge in a couple hours unless I hear concerns. Thanks!

**Pull request merged**
